pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/rule/AbstractApexRule.java

```diff
@@ -106,6 +107,7 @@ public AbstractApexRule() {
         definePropertyDescriptor(CODECLIMATE_CATEGORIES);
         definePropertyDescriptor(CODECLIMATE_REMEDIATION_MULTIPLIER);
         definePropertyDescriptor(CODECLIMATE_BLOCK_HIGHLIGHTING);
+        definePropertyDescriptor(new IntegerProperty("codacy_minutes_to_fix", "Codacy.com Time to fix in minutes", Integer.MIN_VALUE, Integer.MAX_VALUE, 5, 1.0f));
```

Comment:
Property descriptors should be `static final` constants so they can be used programmatically, without needing to be first retrieved by name, to access the typed property value.  See the `PropertySource` interface and `net.sourceforge.pmd.lang.apex.rule.complexity.AvoidDeeplyNestedIfStmtsRule` as an example.